cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -88,6 +89,9 @@ func init() {
 			"If you are the only connection to an 8-core server, then \"-n 8\" is a \n"+
 			"reasonable value.  A lower number is suggested if there are multiple \n"+
 			"clients connecting to the same machine.")
+
+	transcribeCmd.Flags().IntVarP(&maxAlternatives, "maxAlts", "a", 1,
```

Comment:
This is more of a general question.
If there are flags that are only valid for specific values of another flag, should we add a domain to the name here, such as "fmt.timeline.maxAlts"?